lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -292,4 +298,45 @@ module.exports = [
       },
     },
   },
+  {
+    lhr: {
+      requestedUrl: 'http://localhost:10200/perf/animations.html',
+      finalUrl: 'http://localhost:10200/perf/animations.html',
+      audits: {
+        'non-composited-animations': {
+          // Requires compositor failure reasons to be in the trace
+          // https://chromiumdash.appspot.com/commit/995baabedf9e70d16deafc4bc37a2b215a9b8ec9
+          _minChromiumMilestone: 86,
+          score: null,
+          displayValue: '1 animated element found',
+          details: {
+            items: [
+              {
+                node: {
+                  type: 'node',
+                  path: '1,HTML,1,BODY,1,DIV',
+                  selector: 'body > div#animated-boi',
+                  nodeLabel: 'div',
+                  snippet: '<div id="animated-boi">',
+                },
+                subItems: {
+                  items: [
+                    {
+                      failureReason: 'Unsupported CSS Property',
+                    },
+                    {
```

Comment:
feels a bit verbose to put these all as open objects taking 3 lines each, done this way for a particular reason? perhaps future property expansion you have planned?

Reply:
There used to be another property (`animation`) in addition to `failureReason`.

Reply:
It's possible that we will add more properties but I will shorten this for now.